### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by eilandert**
Adding wordpress-hardening-plugin.

Although I run this on a couple of boxes now, I do have to review everything once more. 
Input is much appreciated, since I have just started with modsecurity ;-)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,4 +33,5 @@ maps the regular CRS IDs from 900K for each rule to the range 9,900,000 - 9,999,
 | roundcube-rule-exclusions-plugin    | 9,519,000 - 9,519,999 | https://github.com/EsadCetiner/roundcube-rule-exclusions-plugin          | 3rd party | &#9989;&nbsp;tested |      |
 | sogo-rule-exclusions-plugin         | 9,520,000 - 9,520,999 | https://github.com/EsadCetiner/sogo-rule-exclusions-plugin               | 3rd party | &#9989;&nbsp;tested |      |
 | iredadmin-rule-exclusions-plugin    | 9,521,000 - 9,521,999 | https://github.com/EsadCetiner/iredadmin-rule-exclusions-plugin          | 3rd party | &#9989;&nbsp;tested |      |
-| incubator                           | 9,900,000 - 9,999,999 | https://github.com/coreruleset/incubator-plugin                          | official  | -                   |      |
\ No newline at end of file
+| wordpress-hardening-plugin          | 9,522,000 - 9,522,999 | https://github.com/eilandert/wordpress-hardening-plugin                  | 3rd party | being tested        |      |
```
Comment (theseion) [inline README.md]:
```suggestion
| wordpress-hardening-plugin          | 9,522,000 - 9,522,999 | https://github.com/eilandert/wordpress-hardening-plugin                  | 3rd party | untested        |      |
```

**Review by theseion — CHANGES_REQUESTED** [review]

**Comment by theseion** [comment]
Thank you @eilandert. What is your rational for this plugin? Does it include additional blocking rules (as opposed to rule exclusions)? Should it be installed alongside the `wordpress-rule-exclusions` plugin? The answers to these questions should also be documented in the README of your plugin.

**Comment by dune73** [comment]
Thank you very much. This is very welcome.

We are currently in the final stages of the CRS v4 release. Reviewing and sharing some advice, will probably take a few days and we're very busy. But I really like the initiative.

**Comment by eilandert** [comment]
> Thank you @eilandert. What is your rational for this plugin? Does it include additional blocking rules (as opposed to rule exclusions)? Should it be installed alongside the `wordpress-rule-exclusions` plugin? The answers to these questions should also be documented in the README of your plugin.

The goal is to harden Wordpress a bit more without interfering with it's operation, hopefully our users can drop their security-plugins in Wordpress in the future. It will only contain extra blocking rules since there is already an exclusion-plugin.  Both can be used, in fact it's encouraged to use the exclusions-plugin.  

I have updated the README but will do it in more details soon.

**Comment by eilandert** [comment]
> Thank you very much. This is very welcome.
> 
> We are currently in the final stages of the CRS v4 release. Reviewing and sharing some advice, will probably take a few days and we're very busy. But I really like the initiative.

That's ok, I've got time ;-)

**Review by theseion — APPROVED** [review]

**Comment by theseion** [comment]
Sounds great, thanks!

**Pull request merged**